src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -3202,3 +3202,102 @@ public class CollectionOfTypeWithNonDefaultNamcespace : List<TypeWithNonDefaultN
 
 }
 
+[DataContract]
+public class ObjectContainer
+{
+    [DataMember]
+    private object data;
```

Comment:
The naming convention for private non-static field is to prefix the name with underscore, e.g. _data.